README.md

```diff
@@ -25,6 +25,16 @@ or
 
 `yarn add graphql-hooks`
 
+## Support
+
+- Node.js v8.5+
```

Comment:
I don't think this is accurate, this _should_ run on v4

Reply:
I'll do some local testing

Reply:
As discussed we probably don't need this as we already detail what they need to polyfill, removing.

Reply:
Let's keep this. Nobody should not run on v4 anyway.

Reply:
I've updated this to "Latest 8 & 10 Node releases"